src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java

```diff
@@ -263,6 +263,26 @@ public ResponseEntity<Void> verifyDisputeEmail(@PathVariable(name="id") @Paramet
 		return ResponseEntity.ok().build();
 	}
 
+	@Operation(summary = "Updates the email address of a particular Dispute.")
+	@ApiResponses({
+		@ApiResponse(responseCode = "200", description = "Ok. Email unverified."),
+		@ApiResponse(responseCode = "404", description = "Dispute with specified id not found"),
```

Comment:
I think we should also add 400 bad request as a potential API response.

Reply:
Yes, that's a good call. Missed that, thanks. Will add.